src/app/shared/services/env/env.service.ts

```diff
@@ -12,4 +12,5 @@ export class EnvService {
   kmsServerUrl = undefined;
   stakingPoolFactoryAddress = undefined;
   showAzureLoginOption = true;
+  NATS_ENVIRONMENT_NAME = 'ewf-volta';
```

Comment:
@Passerino is this meant to be used for messaging with `staging` cache-server? If so, can we use `staging` in env variable?
```suggestion
  NATS_ENVIRONMENT_NAME = 'ewf-staging';
```